src/main/java/seedu/address/logic/commands/delete/DeleteExamCommand.java

```diff
@@ -24,28 +23,26 @@
             + "\nExample: " + COMMAND_WORD + " exam 1 "
             + PREFIX_CODE + "CS2103T";
     private final Index targetIndex;
-    private final ModuleCode targetModuleCode;
 
     /**
      * Creates an DeleteExamCommand to delete the Exam at specified {@code Index} of the specified {@code Module}.
      */
-    public DeleteExamCommand(Index targetIndex, ModuleCode targetModuleCode) {
-        requireAllNonNull(targetIndex, targetModuleCode);
+    public DeleteExamCommand(Index targetIndex) {
+        requireAllNonNull(targetIndex);
```

Comment:
can just use `requireNonNull` instead of `requireAllNonNull`. same for delete lesson